Library/Formula/qemu.rb

```diff
@@ -10,6 +10,12 @@ class Qemu < Formula
     sha256 "805da9cd1c6193e345245a4c33dc090fd8b541190c56d64acb927cb37e794d50" => :mountain_lion
   end
 
+  patch do
+    # Fix VENOM
```

Comment:
Done. I think linking to venom.fail should be more then enough, considering that bug is all over the news ;).